README.md

```diff
@@ -115,6 +115,12 @@ The complete cuSignal API documentation including a complete list of functionali
 [cuSignal 0.15 API](https://docs.rapids.ai/api/cusignal/stable/) | [cuSignal 0.16 Nightly](https://docs.rapids.ai/api/cusignal/nightly/)
 
 ## Installation
+### Note: cuSignal installs through Conda will run natively on **ALL** supported GPU architectures.
+
+1. CUDA 10.X -- [60, 61, 62] (Pascal), [70, 72] (Volta), [75] (Turing)
+2. CUDA 11.X -- [60, 61, 62] (Pascal), [70, 72] (Volta), [75] (Turing), [80] (Ampere)
+
+If you need to run on an older compute capability such as Kepler (CC 3.X) or Maxwell (CC 5.X), you will need to build from source. Target architecture will be automatically detected at build time.
```

Comment:
To be clear, will the _entire_ cusignal library need to be installed from source or will users simply need to run `./build.sh` to compile the fatbins alone?

Reply:
Let's add Maxwell back into the source conda build by default and remove these 4 lines of documentation.